AwsCryptographyPrimitives/runtimes/python/src/aws_cryptography_primitives/smithygenerated/aws_cryptography_primitives/deserialize.py

```diff
@@ -0,0 +1,137 @@
+
+async def _deserialize_h_mac(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_HMacOutput(input.value)
+
+async def _deserialize_hkdf_extract(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_HkdfExtractOutput(input.value)
+
+async def _deserialize_hkdf_expand(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_HkdfExpandOutput(input.value)
+
+async def _deserialize_hkdf(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_HkdfOutput(input.value)
+
+async def _deserialize_kdf_counter_mode(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_KdfCtrOutput(input.value)
+
+async def _deserialize_aes_kdf_counter_mode(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_AesKdfCtrOutput(input.value)
+
+async def _deserialize_aes_encrypt(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_AESEncryptOutput(input.value)
+
+async def _deserialize_aes_decrypt(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_AESDecryptOutput(input.value)
+
+async def _deserialize_generate_rsa_key_pair(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
+    return await _deserialize_error(input.error)
+  return aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.dafny_to_smithy.aws_cryptography_primitives_GenerateRSAKeyPairOutput(input.value)
+
+async def _deserialize_get_rsa_key_modulus_length(input: DafnyResponse, config: Config):
+
+  if input.IsFailure():
```

Comment:
nit: PEP8 wants 4 space multiples for indents

https://peps.python.org/pep-0008/#indentation